lib/ui/profile/UserProfile.dart

```diff
@@ -0,0 +1,355 @@
+    color: Color(0xff4c4e6e),
+  );
+
+  final TextStyle detailTextStyle = const TextStyle(
+    fontSize: 13,
+    fontWeight: FontWeight.normal,
+    color: Color(0xffb7b8c9),
+  );
+
+  final TextStyle buttonTextStyle = const TextStyle(
+    fontSize: 13,
+    fontWeight: FontWeight.w500,
+    color: Color(0xff4c4e6e),
+  );
+
+  final EdgeInsets edgePadding = const EdgeInsets.symmetric(horizontal: 32);
+  final EdgeInsets headerEdgePadding = const EdgeInsets.only(
+    left: 32.0,
+    top: 26.0,
+    right: 32.0,
+    bottom: 25.0,
+  );
+
+  final EdgeInsets titleLineSpace = const EdgeInsets.only(bottom: 2);
+
+  final BorderRadius buttonBorderShape = const BorderRadius.all(
+    Radius.circular(8),
+  );
+
+  final double imageSpacing = 20;
+  final double subtitleLineSpace = 6.5;
+  final double buttonLineSpace = 25;
+  final double dividerHeight = 2;
+  final double detailSpacing = 23;
+  final double buttonHeight = 40;
+  final double imageSize = 72;
+
+  final int maxLines = 1;
+
+  const _DefaultStyle({
+    Color buttonColor,
+    TextStyle titleTextStyle,
+    TextStyle subtitleTextStyle,
+    TextStyle detailTextStyle,
+    TextStyle buttonTextStyle,
+    EdgeInsets edgePadding,
+    EdgeInsets headerEdgePadding,
+    BorderRadius buttonBorderShape,
+    EdgeInsets titleLineSpace,
+    double headerElevation,
+    double imageSpacing,
+    double subtitleLineSpace,
+    double buttonLineSpace,
+    double dividerHeight,
+    double detailSpacing,
+    double buttonHeight,
+    double imageSize,
+    int maxLines,
+  });
+}
+
+const UserProfileStyle _defaultStyle = const _DefaultStyle();
+
+class UserProfile extends StatelessWidget {
+  final UserProfileViewModel _viewModel;
+  final UserProfileStyle _style;
+
+  const UserProfile({
+    Key key,
+    UserProfileViewModel viewModel,
+    UserProfileStyle style = _defaultStyle,
+  })  : this._viewModel = viewModel,
+        this._style = style,
+        super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return SingleChildScrollView(
+      child: Column(
+        children: <Widget>[
+          _buildHeader(context),
+          _buildListOfActions(),
+        ],
+      ),
+    );
+  }
+
+  Widget _buildHeader(BuildContext context) {
+    return Column(
+      children: <Widget>[
+        Container(
+          margin: _style.headerEdgePadding,
+          child: Row(
+            mainAxisAlignment: MainAxisAlignment.spaceBetween,
+            crossAxisAlignment: CrossAxisAlignment.center,
+            children: <Widget>[
+              _buildMainTextView(context),
+              SizedBox(width: _style.imageSpacing),
+              _buildPlotImage(_viewModel.image),
+            ],
+          ),
+        ),
+        Padding(
+          padding: _style.edgePadding,
+          child: Column(
+            children: <Widget>[
+              RoundedButton(
+                viewModel: RoundedButtonViewModel(
+                  title: _Strings.SWITCH_PROFILE,
```

Comment:
no snake case... camel case please. (search project for _Strings classes and see)

Reply:
Changed